### PR TITLE
Ocr functions tests

### PR DESCRIPTION
**Pull request by ackaufman**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (seushermsft) [inline .vscode/settings.json]: Maybe we need to add .vscode to .gitignore since its getting included.

Comment (ackaufman) [reply .vscode/settings.json]: Will do.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.vscode
```
Comment (seushermsft) [inline .gitignore]: All this code looks good, but you need to refresh your requirements.txt so the build can succeed.

Comment (ackaufman) [reply .gitignore]: Just saw this comment...actually pretty proud of the fact that I figured that one out on my own before I saw this note!

**Pull request merged**